web_src/js/index.js

```diff
@@ -1831,6 +1832,10 @@ function initAdmin() {
           $('.oauth2_token_url, .oauth2_auth_url, .oauth2_profile_url').show();
           $('#oauth2_email_url').val('');
           break;
+        case 'mastodon':
+          $('.oauth2_auth_url input').attr('required', 'required');
+          $('.oauth2_auth_url').show()
```

Comment:
```suggestion
          $('.oauth2_auth_url').show();
```
frontend lint fails otherwise.